styles/pluralkit/catppuccin.user.less

```
@@ -0,0 +1,495 @@
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Apluralkit
+@description Soothing pastel theme for PluralKit
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("dash.pluralkit.me") {
+  :root.dark {
+    #catppuccin(@darkFlavor);
+  }
+  :root.light {
+    #catppuccin(@lightFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    @rosewater: @catppuccin[@@flavor][@rosewater];
+    @flamingo: @catppuccin[@@flavor][@flamingo];
+    @pink: @catppuccin[@@flavor][@pink];
+    @mauve: @catppuccin[@@flavor][@mauve];
+    @red: @catppuccin[@@flavor][@red];
+    @maroon: @catppuccin[@@flavor][@maroon];
+    @peach: @catppuccin[@@flavor][@peach];
+    @yellow: @catppuccin[@@flavor][@yellow];
+    @green: @catppuccin[@@flavor][@green];
+    @teal: @catppuccin[@@flavor][@teal];
+    @sky: @catppuccin[@@flavor][@sky];
+    @sapphire: @catppuccin[@@flavor][@sapphire];
+    @blue: @catppuccin[@@flavor][@blue];
+    @lavender: @catppuccin[@@flavor][@lavender];
+    @text: @catppuccin[@@flavor][@text];
+    @subtext1: @catppuccin[@@flavor][@subtext1];
+    @subtext0: @catppuccin[@@flavor][@subtext0];
+    @overlay2: @catppuccin[@@flavor][@overlay2];
+    @overlay1: @catppuccin[@@flavor][@overlay1];
+    @overlay0: @catppuccin[@@flavor][@overlay0];
+    @surface2: @catppuccin[@@flavor][@surface2];
+    @surface1: @catppuccin[@@flavor][@surface1];
+    @surface0: @catppuccin[@@flavor][@surface0];
+    @base: @catppuccin[@@flavor][@base];
+    @mantle: @catppuccin[@@flavor][@mantle];
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    body {
+      background-color: @base;
+      color: @text;
+    }
+    .navbar.bg-light {
+      background-color: @base !important;
+    }
+    .navbar-light .navbar-brand {
+      color: @text;
+    }
+    .navbar-light .navbar-bar:hover {
+      color: lighten(@subtext0, 5%);
+    }
+    .navbar-light .navbar-nav .nav-link {
+      color: @subtext0;
+    }
+    .navbar-light .navbar-nav .nav-link:hover {
+      color: lighten(@subtext0, 5%);
+    }
+    .navbar-light .navbar-nav .nav-link:focus {
+      color: lighten(@subtext0, 5%);
+    }
```

Please use nesting here: https://lesscss.org/features/#parent-selectors-feature.
```suggestion
    .navbar-light {
      .navbar-brand {
        color: @text;
      }
      .navbar-bar:hover {
        color: lighten(@subtext0, 5%);
      }
      .navbar-nav .nav-link {
        color: @subtext0;

        &:hover {
          color: lighten(@subtext0, 5%);
        }
        &:focus {
          color: lighten(@subtext0, 5%);
        }
      }
    }
```